pyproject.toml

```diff
@@ -1,47 +1,49 @@
 [tool.poetry]
 name = "nautobot-version-control"
 version = "1.0.0-alpha.0"
-description = "Nautobot Version Control"
+description = "Nautobot Version Control with Dolt"
 authors = ["Network to Code, LLC <opensource@networktocode.com>"]
 license = "Apache-2.0"
-repository = "https://github.com/nautobot/nautobot-plugin-version-control"
-documentation = "https://nautobot-version-control.readthedocs.io"
 readme = "README.md"
+homepage = "https://github.com/nautobot/nautobot-app-version-control"
+repository = "https://github.com/nautobot/nautobot-app-version-control"
+documentation = "https://docs.nautobot.com/projects/version-control/en/latest/"
+keywords = ["nautobot", "nautobot-app", "nautobot-plugin"]
 classifiers = [
     "Intended Audience :: Developers",
-    "Development Status :: 3 - Alpha",
-    "License :: OSI Approved :: Apache Software License",
+    "Development Status :: 5 - Production/Stable",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.7",
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
 ]
 packages = [
     { include = "nautobot_version_control" },
 ]
 include = [
-    "LICENSE.txt",
+    # Poetry by default will exclude files that are in .gitignore
+    "nautobot_version_control/static/nautobot_version_control/docs/**/*",
 ]
 
 [tool.poetry.dependencies]
 python = ">=3.8,<3.12"
-nautobot = "~1.5"
-# MySQL database adapter
+# Used for local development
+nautobot = "^2.0.3"
 mysqlclient = "^2.0.3"
-python-dotenv = "*"
```

Comment:
Are we confident that `python-dotenv` is not a dependency of this App?

Reply:
that should be accurate, I remember removing it

Reply:
We don't need `python-dotenv` now.